app/src/main/java/com/onegini/mobile/exampleapp/network/AnonymousService.java

```diff
@@ -20,9 +20,9 @@
 import com.onegini.mobile.exampleapp.model.ApplicationDetails;
 import com.onegini.mobile.exampleapp.network.client.AnonymousClient;
 import com.onegini.mobile.exampleapp.network.client.SecureResourceClient;
-import rx.Observable;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.schedulers.Schedulers;
+import io.reactivex.Single;
```

Comment:
Since we modify the code in example app, we need to update code snippets in the SDK as well